shared/src/commonMain/kotlin/com/mongodb/dublinmug_kmm/Models.kt

```diff
@@ -1,11 +1,11 @@
 package com.mongodb.dublinmug_kmm
 
 import io.realm.kotlin.types.RealmObject
+import io.realm.kotlin.types.RealmUUID
 import io.realm.kotlin.types.annotations.PrimaryKey
 
 class QueryInfo : RealmObject {
-
     @PrimaryKey
-    var _id: String = ""
+    var _id: String = RealmUUID.random().toString()
```

Comment:
I would consider changing the serverside schema to be a `UUID`, then you can remove the `toString()` here.